src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/CostDetails.js

```diff
@@ -48,6 +46,11 @@ export const CostDetails = ({
     TYPE: getFieldName(55),
   };
 
+  const [setExchangeRateValue, currency] = useFieldMappingFieldValue([SET_EXCHANGE_RATE_FIELD, CURRENCY_FIELD]);
+
+  const [isUseExchangeChecked, setIsUseExchangeChecked] = useState(!isEmpty(setExchangeRateValue));
```

Comment:
`isEmpty(setExchangeRateValue)` can be moved to variable, now it's not clear what this initial state means